lib/bb/tests/fetch.py

```
@@ -323,6 +323,21 @@ class URITest(unittest.TestCase):
             'query': {"9BE0BF6657": None},
             'relative': False
+        },
+        "file://example@.service": {
+            'uri': 'file:example%40.service',
+            'scheme': 'file',
+            'hostname': '',
+            'port': None,
+            'hostport': '',
+            'path': 'example@.service',
+            'userinfo': '',
+            'userinfo': '',
```

Ruff (0.8.2)

Dictionary key literal `'userinfo'` repeated

Remove repeated key literal `'userinfo'`

(F601)